rust/scx_utils/src/topology.rs

```diff
@@ -275,8 +275,13 @@ fn cpus_online() -> Result<Cpumask> {
         let (min, max) = match sscanf!(group.trim(), "{usize}-{usize}") {
             Ok((x, y)) => (x, y),
             Err(_) => {
-                bail!("Failed to parse online cpus {}", group.trim());
-            }
+                match sscanf!(group.trim(), "{usize}") {
+                    Ok(x) => (x, x),
+                    Err(_) => {
+                        bail!("Failed to parse online cpus {}", group.trim());
+                    }
+                }
+            },
         };
         for i in min..max {
```

Comment:
I think we'll also need https://github.com/sched-ext/scx/pull/191/commits/583696f9402f0ae66834b8164c274dc1a65d77f4 right?

Reply:
Just pushed so should be able to just rebase

Reply:
> I think we'll also need [583696f](https://github.com/sched-ext/scx/commit/583696f9402f0ae66834b8164c274dc1a65d77f4) right?

Indeed, thanks for the extra fix! :)